test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -452,6 +452,19 @@ def forward(self, x_in):
         x = {"test_key_in": torch.randn(1, 2, 3)}
         self.run_test(MyModel(), (x,))
 
+    def test_none_as_named_input(self):
```

Comment:
```suggestion
    def test_none_as_input(self):
```
since this supports non-optional input as well. In fact, please try add a test case for it.